### ISSUE TITLE
unable to start the jira service 

### ISSUE THREAD

**Issue by muthuaravind** [alvistack/ansible-role-jira#9]
fatal: [localhost]: FAILED! => {
    "changed": false, 
    "invocation": {
        "module_args": {
            "daemon_reexec": false, 
            "daemon_reload": false, 
            "enabled": true, 
            "force": null, 
            "masked": null, 
            "name": "jira", 
            "no_block": false, 
            "scope": null, 
            "state": "started", 
            "user": null
        }
    }, 
    "msg": "Unable to start service jira: Job for jira.service failed because the control process exited with error code.\nSee \"systemctl status jira.service\" and \"journalctl -xe\" for details.\n"
}
...ignoring
META: ran handlers
META: ran handlers
META: ran handlers

**Comment by hswong3i** [comment]
Sorry for late reply, is it still an active issue? If so please provide more information, e.g.
- `journalctl -xef -u jira.service`
- `tail -f -n 50 /opt/atlassian/jira/logs/catalina.out`
- `tail -f -n 50 /var/atlassian/application-data/jira/logs/atlassian-jira.log`

**Comment by riimak** [comment]
From bin/setenv.sh I had to completely remove the GC_JVM_PARAMETERS so I changed the line from 

CATALINA_OPTS="${GC_JVM_PARAMETERS} ${CATALINA_OPTS}"

to

CATALINA_OPTS="${CATALINA_OPTS}".

After that, service started. But I am wondering, why is that? I even tried placing a openjdk 14 and setting ignore_jvm_version=true but did not help. Do i need some specific version of the openjdk to work with your setup for setenv.sh?

**Comment by riimak** [comment]
In playbook, setting the version to version 8, all good now!
Playbook looks like this now:

`- hosts: all
  vars:
    openjdk_release: "8"
    _jira_download:
      8.4.2:
        url: "https://product-downloads.atlassian.com/software/jira/downloads/atlassian-jira-core-8.4.2.tar.gz"
        dest: "{{ ansible_user_dir }}/.ansible/tmp/atlassian-jira-core-8.4.2.tar.gz"
        checksum: "sha1:7dd2678ed9128bd8a10fae674af69b1e0b9b2c25"
        jira_version: "8.4.2"
    jira_jvm_support_recommended_args: >-
      -Datlassian.plugins.enable.wait=300
  tasks:
    - include_role:
        name: alvistack.python
      tags: python

    - include_role:
        name: alvistack.openjdk
      tags: openjdk

    - include_role:
        name: alvistack.jira
      tags: jira`

**Comment by hswong3i** [comment]
Yes, we are supporting JDK 8 now, duplicated with https://github.com/alvistack/ansible-role-jira/issues/10
